### ISSUE TITLE
Improve Theia's Accessibility

### ISSUE THREAD

**Issue by msujew** [eclipse-theia/theia#13048]
Inspired by [this question](https://community.theia-ide.org/t/theia-accessibility/3260).

I have created an estimate last year to make Theia more accessible, especially so that certain accessibility guidelines are fulfilled or at least so that Theia aligns to vscode in that regard. These were my findings:

**Keyboard navigation**

- Quick navigation across the workbench with (Shift)+F6 missing
- Using tab or Shift+tab to navigate between visible workbench elements (Navigator / Editor / Terminal / etc.) is possible, but needs some polishing
	- Navigating between tabs isn't working too well
	- Tab trapping does work as well, but has no indicator currently

**Screen Reader Capabilities**

- VSCode has aria labels for some elements, especially those which are rendered using markdown
- Completely missing in Theia

**Terminals**

- Contrast ratio cannot be changed, unlike in VSCode

**Sound Cues**

- Completely missing in Theia, they can probably be enabled through some monaco API
